sigmabot.py

```diff
@@ -92,6 +92,11 @@ async def marketcap(ctx):
 async def genesis(ctx):
 	await ctx.send(fq.faq_genesis())
 
+## NFTs
+@client.command()
+async def genesis(ctx):
```

Comment:
@cafebedouin there is already a command called `genesis` and need a `modifier` argument. maybe take an example from `tokenjay` ?